.golangci.yml

```diff
@@ -1,4 +1,13 @@
 linters-settings:
+  gocritic:
+    enabled-checks:
+      - emptyStringTest
+      - evalOrder
+      - paramTypeCombine
+      - preferStringWriter
+      - sprintfQuotedString
+      - stringConcatSimplify
+      - yodaStyleExpr
```

Comment:
I tend to copy paste the descriptions available for each linters.

You can find them in disabled-checks node of .golangci-lint.reference.yml

BTW, maybe I should open a PR to add them to .golangci-lint.reference.yml